client/setup.cfg

```diff
@@ -31,7 +31,7 @@ install_requires =
     pandasql>=0.7.3
     typing_extensions>=4.1.1
     dataclasses==0.6
-    flask==2.2.1
+    flask>=3.0.0
```

Comment:
Should we just pin it to 3.0.0

Reply:
sure

Reply:
actually naw, starts messing with peoples local dependencies when we do that